src/pages/journal/journal.ts

```diff
@@ -230,10 +246,24 @@ export class JournalPage extends BasePageComponent implements OnInit, OnDestroy
     super.logout();
   }
 
-  showTestReportPracticeMode = () :boolean =>
+  showTestReportPracticeMode = (): boolean =>
     this.appConfigProvider.getAppConfig().journal.enableTestReportPracticeMode
 
-  showEndToEndPracticeMode = () :boolean =>
+  showEndToEndPracticeMode = (): boolean =>
     this.appConfigProvider.getAppConfig().journal.enableEndToEndPracticeMode
 
+  updateIncompleteTests = (): number => {
+
+    let testsStatuses;
+    this.pageState.testStatuses$.subscribe((res) => {
```

Comment:
Would have preferred to see the selection of `testStatuses` and `slots` from the store to be abstracted behind the provider which could expose an `Observable<number>`.